<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicsSettingsWidget</class>
 <widget class="QWidget" name="GraphicsSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Renderer</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Renderer:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="renderer">
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Adapter:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="adapter"/>
      </item>
      <item row="2" column="0" colspan="2">
       <layout class="QGridLayout" name="basicCheckboxGridLayout1">
        <item row="0" column="0">
         <widget class="QCheckBox" name="vsync">
          <property name="text">
           <string>Sync To Host Refresh (VSync)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="useBlitSwapChain">
          <property name="text">
           <string>Use Blit Swap Chain</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="hardwareRendererGroup">
     <property name="title">
      <string>Hardware Renderer Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Internal Resolution:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Anisotropic Filtering:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dithering:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="upscaleMultiplier">
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Native (PS2)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2x Native (~720p)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3x Native (~1080p)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4x Native (~1440p/2K)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5x Native (~1620p)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6x Native (~2160p/4K)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7x Native (~2520p)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8x Native (~2280p)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="anisotropicFiltering">
        <item>
         <property name="text">
          <string>Off (Default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8x</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16x</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="dithering">
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Unscaled (Default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Scaled</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Mipmapping:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="mipmapping">
        <item>
         <property name="text">
          <string>Automatic (Default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Basic (Fast)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full (Slow)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>CRC Fix Level:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="crcFixLevel">
        <item>
         <property name="text">
          <string>Automatic (Default)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None (Debug)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Minimum (Debug)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aggressive</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Blending Accuracy:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="blending">
        <item>
         <property name="text">
          <string>None (Fastest)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Basic (Recommended)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium (Debug)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High (Debug)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QGridLayout" name="basicCheckboxGridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="accurateDATE">
          <property name="text">
           <string>Accurate DATE</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="conservativeBufferAllocation">
          <property name="text">
           <string>Conservative Buffer Allocation</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Minimum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Advanced Settings</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="enableHWFixes">
          <property name="text">
           <string>Enable HW Fixes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="gpuPaletteConversion">
          <property name="text">
           <string>GPU Palette Conversion</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="softwareRendererGroup">
     <property name="title">
      <string>Software Renderer Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Extra Rendering Threads:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="extraSWThreads">
        <property name="suffix">
         <string> threads</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="swAutoFlush">
          <property name="text">
           <string>Auto Flush</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>Edge Anti-Aliasing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>Mipmapping</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
